tasks.go

```diff
@@ -215,7 +216,11 @@ func RunAllAnalysisTasks(runner Runner, path string, workers int) {
 	log.Printf("Run %d analysis tasks in %v.", taskCount, delta)
 }
 
-func GetAllAnalysisTasks(runner Runner, path string, results chan<- CheckResults) <-chan Task {
+// GetAllAnalysisTasks returns a channel of all tasks known to the dump tool. It returns
```

Comment:
AFAIU it doesn't return "_all tasks_ known to the dump tool", I think tasks were split between dump and analysis?